mypy/stubtest.py

```diff
@@ -367,8 +371,8 @@ def _belongs_to_runtime(r: types.ModuleType, attr: str) -> bool:
             m
             for m in dir(runtime)
             if not is_probably_private(m)
-            # Ensure that the object's module is `runtime`, since in the absence of __all__ we
-            # don't have a good way to detect re-exports at runtime.
+            # Filter out objects that originate from other modules (best effort). Note that in the
+            # absence of __all__, we don't have a way to detect re-exports at runtime
```

Comment:
Nit: I'd prefer "surefire way" or "perfect way" here, since arguably we do have _a_ way of detecting re-exports (using `symtable` to parse the source code), it just isn't 100% reliable (doesn't work for `*` imports; and you can't always get the source code)

Reply:
I guess I meant to say "explicit re-export". symtable doesn't tell you if something is `import X as X` right?

Reply:
Added "explicit", I think that gets us on the same page :-)

Reply:
Ahh gotcha, yup! LGTM now :)